src/views/HomeView.vue

```diff
@@ -1,7 +1,30 @@
 <template>
     <HelloWorld />
+
+    <h2>
+        Voor debug mogelijkheden kan hier ingelogd worden met verschillende
+        accounts
+    </h2>
+    <v-btn class="mx-2" @click="login(1)">Login als user Jan Bakker</v-btn>
+
+    <v-btn @click="test()">test</v-btn>
+    <br />
+    {{ whoami }}
 </template>
 
 <script lang="ts" setup>
+import API from "@/Api";
 import HelloWorld from "@/components/HelloWorld.vue";
+import { useSessionStore } from "@/store/sessionStore";
+import { ref } from "vue";
+const sessionStore = useSessionStore();
+const whoami = ref({});
+async function login(name: number) {
```

Comment:
je verwacht name, maar geeft number. Je verwacht id denk ik?

Reply:
nee, we hebben hardcoded users die een nummer in de naam hebben

Reply:
dank u, 010